xtask/src/build_docker_image.rs

```diff
@@ -355,25 +358,29 @@ pub fn build_docker_image(
     Ok(())
 }
 
-fn is_armv6(target: &str) -> bool {
+fn is_armv6(target: &ImageTarget) -> bool {
     matches!(
-        target,
+        target.name.as_str(),
         "arm-unknown-linux-gnueabi" | "arm-unknown-linux-musleabi"
     )
 }
 
-fn is_armv7(target: &str) -> bool {
+fn is_armv7(target: &ImageTarget) -> bool {
     matches!(
-        target,
+        target.name.as_str(),
         "armv7-unknown-linux-gnueabihf" | "armv7-unknown-linux-musleabihf"
     )
 }
 
```

Comment:
My bad! I meant more that these could live in a `impl ImageTarget`

Reply:
Ah that makes sense. That's a good idea, I'll move them in then.